### ISSUE TITLE
EDDI disabling Edit and Delete buttons after recovering a script

### ISSUE THREAD

**Issue by Darkcyde13** [EDCD/EDDI#2459]
### Expected
When EDDI recovers a script after a crash, that you can edit or delete that script.

### Observed
EDDI disables the Edit and Delete buttons for the script until you restart EDDI.

### Steps to reproduce
1. Edit a script in EDDI, either stand alone, or via Voice Attack.
2. Cause EDDI to crash (see #2458 for a way to do this).
3. Reopen EDDI, either standalone or via VA.
4. At the prompt to reopen the script that was being edited, say yes.
5. Close the script window by clicking OK.
6. Observe the Edit and Delete buttons are greyed out and not selectable.

### Configuration
* Version [Please give full version numbers here, not just "latest" or "current"]
  * EDDI Version: v4.0.2-b4
  * VoiceAttack Version: v1.10.3
* Localization [if applicable]
  * EDDI's language: [or the operating system language if EDDI's language setting is "Automatic"]
  * Voice: [Name and language]

### Investigation Notes
While looking into the problem with the System scan complete script (#2458) I encountered this problem.  I had seen it once before but as EDDI doesn't crash very often (for me at least) while I have a script open, I forgot about  it.

Following the steps above with the problem from #2458, I can consistently cause this bug to occur.  When clicking OK on the script, any changes that were made prior to the crash, or after the recovery, are not saved.

To clear it, you have to close and restart EDDI or VA without the offending script open.

If you say no when the prompt pops up, then the buttons are useable.

### EDDI Logs
N/A

### Player journals
N/A

**Comment by Darkcyde13** [comment]
Today I can't make System scan complete crash at all, so I'm not sure how you'll be able to test this properly now.  Although, I guess you have a way to test things like this anyway. 🙂

**Comment by Darkcyde13** [comment]
@Tkael I'm not sure if this is related to another similar problem I've been experiencing, but thought I'd add it here just in case.

Sometimes, when I add a new script, the Edit and Delete buttons are greyed out, just like when it was for recovering a script after a crash.  It doesn't seem to happen all the time, so I'm not really sure what criteria make it happen.  I'll try to make note of what I do when/if it happens again.  I'm reporting it now, because it just happened, but I'm not sure what sequence of events led up to it.  I know I had swapped personalities between my work-in-progress and the EDDI default a couple of times, then made a new test script, and when I clicked OK, the buttons were greyed out.  So it's possible this is what caused it, swapping between a user personality, the EDDI default, then back again.
